cli/cmd/generate_gcp.go

```diff
@@ -29,6 +30,11 @@ var (
 	QuestionExistingServiceAccountName       = "Specify an existing service account name:"
 	QuestionExistingServiceAccountPrivateKey = "Specify an existing service account private key (base64 encoded):"
 
+	GcpAdvancedOptAgentless      = "Configure additional Agentless options"
+	QuestionGcpProjectFilterList = "Specify a comma seprated list of Google Cloud projects that " +
+		"you want to monitor: (optional)"
+	QuestionGcpRegions = "Specify a comma seprated list of regions to deploy:"
```

Comment:
Is this actually an advanced option? Seems like the generate command fails if this is not populated.

Reply:
Good call. Moved this question to the top level so users have to answer it.